examples/resources/umaa_components.xml

```diff
@@ -15,22 +15,22 @@ to use the software.
 <dds xsi:noNamespaceSchemaLocation="http://community.rti.com/schema/current/rti_dds_profiles.xsd"
      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
 
-    <!-- For 6.1.2.14+ can set default variable here or can set as Environment Variable  
-         Use ENV variables with the syntax: $(ANCHOR_CONTROLLER_GUID)                               
+    <!-- You can assign ENV variables in 3 ways to use within this file.
+     - Using the <configuration_variables> tag for defaults (For Connext > 6.1.2.14+)
+     - Using command line i.e. -DANCHOR_CONTROLLER_GUID=05 05 05 05 05 05 05 05 05 05 05 05 05 05 05 05
+     - Using a bash script i.e. export ANCHOR_CONTROLLER_GUID=05 05 05 05 05 05 05 05 05 05 05 05 05 05 05 05

```

Comment:
I think that we need to add quotes between the value that contains spaces in bash. Potentially the same for the command line option